.github/workflows/crossplane-consumer-validation.yaml

```diff
@@ -35,10 +35,10 @@ jobs:
           CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.sha }})
 
           # Categorize consumer changes
-          COMPOSITE_RESOURCES=$(echo "$CHANGED_FILES" | grep -E 'control-plane-crossplane/workload-clusters/.*\.yaml$|control-plane-crossplane/platform-services/.*\.yaml$' | grep -v kustomization.yaml || true)
+          COMPOSITE_RESOURCES=$(echo "$CHANGED_FILES" | grep -E 'bootstrap/control-plane/crossplane/workload-clusters/.*\.yaml$|bootstrap/control-plane/crossplane/platform-services/.*\.yaml$' | grep -v kustomization.yaml || true)
```

Comment:
current implementation of "bootstrap" does not align with the design by hosting manifests which are meant for permanent control-plane cluster. We will need to update it later (added as review now)